packages/core/app/fs-next-update/page.tsx

```diff
@@ -1,3 +1,63 @@
-export default function Page() {
-  return <div>placeholder</div>
+import type { PageContentType } from 'src/server/cms'
+import { getPage } from 'src/server/cms'
+import CUSTOM_COMPONENTS from 'src/customizations/src/components'
+
+import { getDynamicContent } from 'src/utils/dynamicContent'
+import storeConfig from '../../faststore.config'
+import { ComponentType } from 'react'
+import { OverriddenDefaultHero as Hero } from 'src/components/sections/Hero/OverriddenDefaultHero'
+import PageProvider from 'src/sdk/overrides/PageProvider'
+import RenderSections from 'app/components/cms/RenderSections'
```

Comment:
I think we should migrate all these files from the `src` folder to the `app` folder, except the `src/customizations`, so that we can leverage the RSC.

Reply:
I don't think the file needs to be in the app folder to be a server component. I think it just needs to be called within the app folder. Many projects are leaving only the routes and files related to the routes inside the app folder and the rest outside.
Example here: https://github.com/joschan21/casecobra/blob/master/src/components/Navbar.tsx

Reply:
I move several dependencies to app folder.